drivers/src/mailbox.rs

```diff
@@ -347,6 +347,19 @@ impl MailboxRecvTxn<'_> {
         Ok(())
     }
 
+    /// Pulls at most `count` words from the mailbox and throws them away
+    pub fn drop_words(&mut self, count: usize) -> CaliptraResult<()> {
```

Comment:
create an issue to add a unit test for this later?

Reply:
Added a unit test